scss/partials/_cards.scss

```diff
@@ -124,11 +124,15 @@
       @extend [class^="tertiary-info"];
       color: $secondary-green-1;
       margin-bottom: 1rem;
-  
+
       @include breakpoint(tablet) {
         margin-bottom: 1.5rem;
       }
     }
+
+    &-blog {
```

Comment:
Is this meant to be a single hyphen? If it's a modifier, it should be a double hyphen.

Reply:
Good catch!